### ISSUE TITLE
searching the website for mysqli functions does not lead to a result

### ISSUE THREAD

**Issue by staabm** [php/web-php#677]
search php.net for e.g. `mysqli_fetch_row` does not find a corresponding manual page

see https://www.php.net/manual-lookup.php?pattern=mysqli_fetch_row&scope=quickref

**Comment by cmb69** [comment]
Yeah, that's a long standing issue: https://bugs.php.net/bug.php?id=49956. Unfortunately, not quite easy to resolve.

**Comment by bxmpublishingonline** [comment]
If  anyone is getting WP installs failing due to mysqli not being found or your site is now 404 error loading then go back 8.0.

I noticed when using 8.1 websites would get a 404 error and some applications like WP would fail install due to mysqli not being found. going back to 8.0 fixed the issue for me. mysqli won't even show in the extensions tab when using 8.1.

This is using cPanel with Verpex Hosting.

**Comment by Roy-Orbison** [comment]
This is semi-resolved since #1007 was merged. Typing `mysqli_fetch_row` in the search box produces `mysqli_result::fetch_row` as the second autocomplete result, though submitting it (without selecting a suggestion) as a server-side search still does not find it.

**Comment by saundefined** [comment]
@staabm Hi, after updating the search, I guess the issue is no longer relevant, and we can close it.
